src/components/Block.tsx

```diff
@@ -0,0 +1,140 @@
+import clsx from "clsx";
+import React, { FC, KeyboardEvent, useState } from "react";
+import { ActionsProps } from "./Actions";
+import { Card, CardProps } from "./Card";
+import { List, ListProps } from "./List";
+import { Pills, PillsProps } from "./Pills";
+import { Seperator } from "./Seperator";
+
+export interface BlockProps {
+  children: React.ReactElement<ListProps | PillsProps | CardProps> | React.ReactElement<ListProps | PillsProps | CardProps>[];
+  actions?: React.ReactElement<ActionsProps>;
+  imagePath?: string
+  title?: string
+  subtitle?: string
+  size: number
+  isEditable: boolean
+  handleTitleChange?: (title: string) => void
+  handleSizeChange?: (size: number) => void
+}
+
+export const Block = ({children, actions, imagePath, title, subtitle, size, isEditable, handleTitleChange, handleSizeChange}: BlockProps) => {
+
+  const [titleState, setTitleState] = useState<string | undefined>(title);
```

Comment:
@NilsJacobsen How does the title get stored by the database or why is this a local state?

Reply:
It's a controlled input field therefore we need a store here. OnEnter and onBlur we store it in the recoil store and then immediately call the mutation through a function that we provide in the studio. Saw no other way of doing that since I can not call this function from our ui lib in storybook. But I feel like that is pretty understandable for dev. It's gonna become a one-liner in the block.tsx file.